project/AppDependencies.scala

```diff
@@ -6,7 +6,7 @@ object AppDependencies {
   val compile: Seq[ModuleID] = Seq(
```

Comment:
Would be worth to also update the test dependencies?
```
    "uk.gov.hmrc"            %% "bootstrap-test-play-28"   % bootstrapPlayVersion % "test",
    "org.mockito"            %% "mockito-scala"            % "1.17.12"            % "test",
```
Not sure, but I think in that case you can also remove `scalatestplus-play` and `flexmark-all`